source/3dsmethod.rst

```diff
@@ -17,7 +17,7 @@ If `3DS Method URL`_ is included in the :ref:`preauth-usage` response, the
 3DS method *must* be invoked as explained in this guide.
 
 If ``threeDSMethodURL`` *is not* included in the ``/preauth`` response (ref.
-:ref:`2.1.0 <preauth-response-210>`, :ref:`2.1.0 <preauth-response-220>`),
+:ref:`2.1.0 <preauth-response-210>`, :ref:`2.2.0 <preauth-response-220>`, :ref:`2.3.1 <preauth-response-231>`),
```

Comment:
Should probably omit `2.1.0` since it will be gone by the time this documentation is released?

Reply:
Given the size of this PR, i think it will make sense to remove 2.1 in a separate PR

Reply:
Fair enough